activities/live-streaming/index.md

```diff
@@ -0,0 +1,55 @@
+---
+type: index
+---
+
+# Live streamed podcast
+
+We use live streams as a tool to communicate. The format we have chosen is a podcast style live stream and we call the show **Let's talk about Public Code**.
+
+## Goals
+
+- **Become discoverable:** Create another way for people we would like to know who don't know that we exist or what we do *to discover us*.
+- **Grow** our codebases **community**.
```

Comment:
I think this should be either:
singular - codebase's community
plural - codebases' communities
and probably the plural?